lib/twitter_web/templates/browse/index.html.eex

```diff
@@ -0,0 +1,22 @@
+<h1>Listing Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>body</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for post <- @posts do %>
+    <tr>
+      <td><%= post.body %></td>
+
+      <td>
+        <%# <span><%= link "Like", to: Routes.like_path(@conn, :create, post) %></span> %>
```

Comment:
is this commented out? maybe we should delete?

Reply:
I changed the route, but I didn't fix the link. I thought I don't need to fix it here since I was going to do the json files.

Reply:
Usually it is considered bad practice to keep commented out code in a codebase, so it is best to remove it before submitting a PR 🙂

Reply:
ok, fixed